internal/pipeline/file_receiver.go

```diff
@@ -186,6 +186,7 @@ func contains(err error, options ...string) bool {
 }
 
 func (fr *FileReceiver) processMessage(msg *pubsub.Message) error {
+	msg.Ack()
```

Comment:
We used to ack this early and it caused a bunch of issues. We'd silently ignore invalid events and skip events during crashes/shutdowns. 

https://github.com/moov-io/achgateway/commit/b6d63fc12322b2c35377ef8778ae9784aa2f1236 and https://github.com/moov-io/achgateway/commit/d087a2b2d0257f9ebd263043bdae7a83ff5490ff were previous changes around this.